airbyte-workers/src/main/java/io/airbyte/workers/temporal/TemporalClient.java

```diff
@@ -115,10 +141,154 @@ public TemporalClient(final WorkflowClient client, final Path workspaceRoot) {
             connectionId));
   }
 
+  public void submitConnectionUpdaterAsync(final UUID connectionId) {
+    log.info("Starting the scheduler temporal wf");
+    final ConnectionUpdaterWorkflow connectionUpdaterWorkflow = getWorkflowOptionsWithWorkflowId(ConnectionUpdaterWorkflow.class,
+        TemporalJobType.CONNECTION_UPDATER, "connection_updater_" + connectionId);
+    final BatchRequest signalRequest = client.newSignalWithStartRequest();
+    final ConnectionUpdaterInput input = new ConnectionUpdaterInput(connectionId, null, null, false, 1, null);
+    signalRequest.add(connectionUpdaterWorkflow::run, input);
+
+    final ExecutorService threadpool = Executors.newCachedThreadPool();
```

Comment:
Seems like the number of executor services in memory will grow the longer the server is up since these will be created every call and never cleaned up after the task completes? Can we use a shared thread pool for the client?

Reply:
Done